File: pinyougou_manager_web/src/main/webapp/js/service/specificationService.js
app.service("specificationService", function ($http) {

    //条件分页查询
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post("../specification/search.do?pageNum=" + pageNum + "&pageSize=" + pageSize, searchEntity);
    }

    //新增
    this.add = function (entity) {
        return $http.post("../specification/add.do", entity);
    };
    //修改
    this.update = function (entity) {
        return $http.post("../specification/update.do", entity);
    };

    //修改前查询
    this.findOne = function (id) {
        return $http.get("../specification/findOne.do?id=" + id);
    };

    this.dele = function (ids) {
        return $http.get("../specification/dele.do?ids=" + ids);
    };

});

File: pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSpecification;
import entity.PageResult;
import groupentity.Specification;

import java.util.List;
import java.util.Map;

public interface SpecificationService {

    /**
     * 条件分页查询
     */
    public PageResult search(TbSpecification specificationCon, Integer pageNum, Integer pageSize);

    /**
     * 新增
     */
    public void add(Specification specification);

    void update(Specification specification);

    Specification findOne(Long id);

    void delete(Long[] ids);

    List<Map> selectSpecList();
}
